src/main/java/org/miracum/etl/fhirtoomop/mapper/EncounterInstitutionContactMapper.java

```diff
@@ -346,8 +351,7 @@ private VisitOccurrence createNewVisitOccurrence(
             .fhirIdentifier(encounterSourceIdentifier)
             .visitEndDatetime(visitEndDateTime)
             .visitEndDate(visitEndDateTime.toLocalDate())
-            .visitSourceValue(visitSourceValue == null ? null : visitSourceValue.substring(4))
-                .careSiteId(Math.toIntExact(careSiteId))
+            .visitSourceValue(visitSourceValue)
```

Comment:
remove this change. when #8 is merged you can take pull and test this PR. and make changes accordingly if any changes are required

Reply:
This can be removed after merging that PR.